core/javaAction/Dockerfile

```diff
@@ -1,38 +1,23 @@
-FROM buildpack-deps:trusty-curl
+FROM adoptopenjdk/openjdk8-openj9:x86_64-ubuntu-jdk8u162-b12_openj9-0.8.0
```

Comment:
As I mentioned in my comment, use the `jdk8u162-b12_openj9-0.8.0` tag rather than the `x86_64-ubuntu-jdk8u162-b12_openj9-0.8.0`